themes/bootstrap5/templates/tag/userlist.phtml

```diff
@@ -0,0 +1,97 @@
+<?php
+  // Set up breadcrumbs:
+  $this->breadcrumbs()->set($this->translate('Your Account'), $this->url('myresearch-home'))
+    ->add($this->translate('Tags'), active: true);
+
+  $this->assetManager()->appendScriptLink('requests.js');
+  $tabs = $this->accountCapabilities()->getUserContentTabs();
+?>
+<?php $this->headTitle($this->translate('Tags'));?>
+<?=$this->component('show-account-menu-button')?>
+
+<div class="<?=$this->layoutClass('mainbody')?>">
+  <ul class="nav nav-tabs" role="tablist">
+    <?php foreach ($tabs as $tab): ?>
+      <li class="nav-item" role="presentation" id="user-content-<?=$tab?>" data-tab="<?=$this->escapeHtmlAttr($tab)?>"><a class="<?='tag' === $tab ? 'active' : ''?> nav-link" href="<?=$this->url($tab . '-' . 'userlist')?>"><?=$this->transEsc(ucfirst($tab))?></a></li>
```

Comment:
Is there a reason that you escape `$tab` in `data-tab` but not in `id`? I think it would make sense to be consistent.